packages/components/src/DataList/context/DataListContext/DataListContext.tsx

```diff
@@ -6,6 +7,9 @@ export const defaultValues = {
   data: [],
   headers: {},
   children: [],
+  actions: undefined,
+  onSelectChange: noop,
```

Comment:
this prop is optional so regardless of it having a `noop` or not being defined here, typescript will still force you to guard it. In short, don't really need to do this